pages/search.js

```diff
@@ -142,7 +142,7 @@ class SearchPage extends React.Component {
       isHost: isNil(query.isHost) ? undefined : parseToBoolean(query.isHost),
       country: query.country || null,
       sortBy: query.sortBy || 'ACTIVITY',
-      tag: query.tag?.split(',') || [],
+      tag: query.tag?.length > 0 ? query.tag.split(',') : [],
```

Comment:
No blocker but I'm curious about this change, is it just code style or is it solving something?